Add tutorials for ApplyForceTorque and MouseDrag plugins (#2083)
gazebosim/gz-sim: tutorials/apply_force_torque.md

> \page apply_force_torque Apply Force and Torque

Reviewer: I think you'll need to add a link to this page in https://github.com/gazebosim/gz-sim/blob/gz-sim7/tutorials.md.in

order of milliseconds, thus the need for such a large force.

![Apply Force](https://raw.githubusercontent.com/gazebosim/gz-sim/main/tutorials/files/apply_force_torque/Force.png)

### Apply torque to a link

On the dialog, write `2000` on the `X` field under `Torque` and click on
`Apply Torque` to see the cylinder rotate slightly.

![Apply Torque](https://raw.githubusercontent.com/gazebosim/gz-sim/main/tutorials/files/apply_force_torque/Torque.png)

### Apply force with an offset

By default, the force is applied to the link's center of mass, but this can be
modified through the `Offset` fields. On the dialog, write `1000` on the `X`
field under `Force` and `1` under the `Z` field under `Offset`. Press
`Apply Force` to see the model move slightly in the `X` direction while also
rotating around the `Y` direction.

![Apply Force Offset](https://raw.githubusercontent.com/gazebosim/gz-sim/main/tutorials/files/apply_force_torque/ForceOffset.png)

### Rotation tool

On the dialog, write `10000` on the `X` field under `Force`. Click on the force
vector to make the rotation tool appear. Drag the blue circle to rotate the

> force so that it is alligned with the `Y` direction. Notice how the XYZ fields

Reviewer: ```suggestion
force around the `Z` axis so that it is aligned with the `Y` direction. Notice how the XYZ fields
```

changed, but not the magnitude. Press `Apply Force` to see the model move in the
`Y` direction.

![Rotation tool](https://raw.githubusercontent.com/gazebosim/gz-sim/main/tutorials/files/apply_force_torque/RotationTool.png)

## The interface explained

> **Note**: If you apply force and/or torque while the simulation is paused,
they will accumulate and be applied all at once when the simulation is
unpaused.

### Force

- **Force X, Y, Z**: Each field specifies how much force will be applied on that

> direction, in N. The frame is fixed to the link.

Reviewer: ```suggestion
direction, in Newtons (N). The frame is fixed to the link.
```

- **Mag**: The total magnitude of the force which will be applied, which is the
Euclidean norm of the 3 forces above. Changing the magnitude changes the XYZ
fields proportionally, maintaining the force direction.

- **Offset X, Y, Z**: By default, force is applied to the link's center of mass,
in meters. Here you can edit the X, Y and Z fields to give the force an offset
with respect to the center of mass expressed in the link's frame.

>   - **Tip**: Right-click the model and choose View -> Center of mass to see its

Reviewer: ```suggestion
  - **Tip**: Right-click the model and choose `View` -> `Center of Mass` to see its
```

>     position. You might want to also make the model transparent for that.

Reviewer: ```suggestion
    position. You will want to also make the model transparent to see the center of mass visual (`View` -> `Transparent`).
```

- **Apply Force**: Click this to apply only force for one time step. Keep in
mind that time steps are in the order of milliseconds, so relatively large
forces are needed in order to apply a significant impulse.

### Torque

- **X, Y, Z**: Each field specifies how much torque will be applied about that

> axis, in N.m. The frame is fixed to the link.

Reviewer: ```suggestion
axis, in Newton-meters (N.m). The frame is fixed to the link.
```

\page apply_force_torque Apply Force and Torque

The Apply Force Torque plugin allows users to apply forces and/or torques to
links in the simulation through the graphical user interface.

## Examples

Let's go through an example of applying force and torque to simple models. Open

> the `shapes` world with

Reviewer 2: ```suggestion
the `shapes.sdf` world with:
```

```bash
gz sim shapes.sdf
```

From the plugin dropdown, select the `Apply Force Torque` plugin. Make sure the
simulation isn't paused.

> ![Interface](https://raw.githubusercontent.com/gazebosim/gz-sim/main/tutorials/files/apply_force_torque/Interface.png)
> 
> 
> ### Apply force to a link

Reviewer 2: ```suggestion
![Interface](https://raw.githubusercontent.com/gazebosim/gz-sim/main/tutorials/files/apply_force_torque/Interface.png)

### Apply force to a link
```

Author: Done in f977a8c
